examples/skinning/src/lib.rs

```diff
@@ -0,0 +1,170 @@
+use std::{path::Path, sync::Arc};
+
+const SAMPLE_COUNT: rend3::types::SampleCount = rend3::types::SampleCount::One;
+
+#[derive(Default)]
+struct SkinningExample {
+    loaded_model: Option<rend3_gltf::LoadedGltfScene>,
+    directional_light_handle: Option<rend3::types::DirectionalLightHandle>,
+}
+
+/// Locates an object in the node hierarchy that corresponds to an animated mesh
+/// and returns its list of skeletons. Note that a gltf object may contain
+/// multiple primitives, and there will be one skeleton per primitive.
+pub fn find_armature<'a>(
+    nodes: impl Iterator<Item = &'a rend3_gltf::Labeled<rend3_gltf::Node>>,
+) -> Option<rend3_gltf::Armature> {
+    for node in nodes {
+        if let Some(ref obj) = node.inner.object {
+            if let Some(ref armature) = obj.inner.armature {
+                return Some(armature.clone());
+            }
+        }
+        if let Some(skels) = find_armature(node.inner.children.iter()) {
+            return Some(skels);
+        }
+    }
+    None
+}
```

Comment:
I think there is potential to add more functions like this to rend3-gltf, but I'll leave that for a later PR.